### PR TITLE
DDF-3757 audit any unsolicited session ID requests

### PR DESCRIPTION
**Pull request by stustison**
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@troymohl 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@bdeining
@millerw8
@pklinef
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

File: platform/security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java [hunk 1]
```diff
@@ -133,6 +134,9 @@ public HandlerResult getNormalizedToken(
 
     HttpSession session = httpRequest.getSession(false);
     if (session == null && httpRequest.getRequestedSessionId() != null) {
+      if (httpRequest.isRequestedSessionIdValid()) {
+        SecurityLogger.audit("Request has requested an invalid session ID for this server.");
```
Comment (troymohl) [inline platform/security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java]: nit: I wonder if `Incoming HTTP Request contained invalid session ID for this server` might make a better message.

Comment (stustison) [reply platform/security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java]: Sure

**Comment by stustison** [comment]
build now

**Comment by stustison** [comment]
build now

**Pull request merged**